server/routes.js

```diff
@@ -28,3 +28,4 @@ r.post('/api/v1/login', account.login);
 r.get('/api/v1/user', account.get);
 r.get('/api/v1/logout', account.logout);
 r.post('/api/v1/updateImage', account.image);
+r.post('/api/v1/user/update', account.update);
```

Comment:
I'm going to change this to a put route, because you're updating fields and returning the complete updated user

Reply:
I need to return the complete updated user though. You can change it to a put (just to make it appear more RESTful), but it makes sense to grab whatever the source of truth is after updating it. You could simply send a success message after updating the document and use the values that were just submitted on the client side to then update the redux state, but it seems clearer and less buggy to have more of a circular motion with client => mongo => client to keep everything in synch. If there is anything we don't want to return from the updated user back to the client, just simply set those property's values to undefined like you were doing. 

Open to discussion on this because I could be very wrong honestly.

Reply:
Sorry I meant that it just needs to be changed from post to put/patch to be more restfull